tests/ui/loginPageUITests.js

```diff
@@ -1,32 +1,40 @@
-require('../../utilities/seleniumKeepAlive');
+require("../../utilities/seleniumKeepAlive");
 
 module.exports = {
-  '@tags': ['parallel'],
-  'Verify nav elements present'(browser) {
+  "@tags": ["parallel"],
+  "Verify nav elements present"(browser) {
     const loginPage = browser.page.loginPage();
-    const pageNav = loginPage.section.navigation;
     loginPage.loadPage();
-    pageNav.waitForElementVisible('@signupLink', browser.globals.elementTimeout);
-    pageNav.expect.element('@logo').to.be.visible;
+    loginPage.waitForElementVisible(
+      "@signupLink",
+      browser.globals.elementTimeout
+    );
+    loginPage.expect.element("@logo").to.be.visible;
   },
-  'Verify login Form elements present'(browser) {
+
+  "Verify Login Form elements present"(browser) {
     const loginPage = browser.page.loginPage();
-    const loginForm = loginPage.section.loginForm;
     loginPage.loadPage();
```

Comment:
Good catch with the fullscreen! this function should include the browser fullscreen window command I think so you're not repeating yourself throughout the tests. Meaning this should load the page and work with it in fullscreen the entire time. This will make sure we're not inadvertently loading the "mobile" view when we are testing as well.

Reply:
Removed